stackle_api/app/routes/api_v1/misc.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Created by bhavyaagg on 18/02/18.
+ */
+
+
+// The routes present here has to be sorted into meaningful routes
+
+const express = require('express');
+const router = express.Router();
+
+const postModels = require('../../models/post')
+const Stack = require('../../models/stack');
+const User = require('../../models/user');
+
+const Post = postModels.Post;
+const Comment = postModels.Comment;
+
+//api
+router.get('/login', function (req, res) {
+})
+
+//comment on a post
+router.post('/comment/:postid', function (req, res) {
+    let postid = req.params.postid;
+    let query = {_id: postid};
+    let comment = new Comment(req.body);
+    Post.update(query, {comments: []});
+})
+
+
+
+//delete stack
+router.delete('/delete/stack/:stackid', function (req, res) {
+    let stack_id = req.params.stackid;
+    Stack.remove({_id: stack_id}, function (err, success) {
+        if (err) {
+            res.send("Couldn't delete Stack");
+        } else {
+            res.send("")
+        }
+    })
+})
+
+//user subscribing to an stack
+router.post('/subscribe', function (req, res) {
+    let userid = req.body.uid;
+    let stackname = req.body.stack_name;
+    let query = {userId: userid};
+    User.findOneAndUpdate(query, {$push: {subscribed_stacks: stackname}}, function (err, noaffected) {
```

Comment:
camel case

Reply:
Will have to change the whole schema. Will update the models in a different PR as a lot of changes would be there.